assets/app/view/game/alternate_corporations.rb

```diff
@@ -510,6 +510,67 @@ def render_rs_income
           h(:div, { style: { gridColumnStart: 2 } }, [movement_table]),
         ])
       end
+
+      # This is used by 1858 which has private railway companies that act a bit
+      # like minor railways, laying track in operating rounds. This rendering
+      # code is a simplified version of Engine::View::Game::Company.render.
+      def render_private_railway
```

Comment:
why is this needed? could you just not having this extra UI?

Reply:
This is needed because the private companies in 1858 are operating entities, so get shown in operating rounds and as entities owned by the player. Without this alternate view they will be rendered as if they were minors, but that's a bit of a mess, you get shown their operating order (which never varies) and that the value of their last run (always zero, as they can't own trains). Plus there's an attempt to put in the full name of the entity on the charter, which doesn't fit.

Here's what this looks like with the default rendering:
![render-minors](https://user-images.githubusercontent.com/11144854/207150046-0b9bdf63-cb4b-4353-a024-f41a5b99605e.png)

This alternate rendering mimics the default view of a private company, which is what these entities are.
![render-privates](https://user-images.githubusercontent.com/11144854/207150070-822acec9-8706-464d-939f-dc644dd0ec0a.png)

Reply:
i think this PR would be a lot simpler if you just made the companies minors

Reply:
OK. I'll look into doing that.